src/bootupd.rs

```diff
@@ -715,6 +755,41 @@ fn strip_grub_config_file(
     Ok(())
 }
 
+/// Determine whether the necessary bootloader files are present for GRUB.
+fn has_grub(source_root: &openat::Dir) -> bool {
+    source_root.open_file(bios::GRUB_BIN).is_ok()
```

Comment:
this causes build error on arch other than powerpc64 or x86_64 due to arch specific use of bios crate on line 1.

Reply:
As a temporary workaround, I've hardcoded the GRUB_BIN value in the function.  Should compile on other architectures now

Reply:
why isn't there a "thank you" reaction emoji?
thank you.